src/bin/console/console.cpp

```diff
@@ -402,7 +403,9 @@ int Console::load_table(const std::string& args) {
     }
   } else if (extension == "tbl") {
     try {
-      auto table = opossum::load_table(filepath, Chunk::MAX_SIZE);
+      const auto default_chunk_size = 500000u;
```

Comment:
If have to use literals, please doc why you chose this specific number

Reply:
Our style guide does not mention it, but do we have an opinion on using C++14's digit separators? Would love to see them here and anywhere.

Reply:
I <3 them.